pom.xml

```diff
@@ -245,10 +246,16 @@
                 <configuration>
                     <modules>
                         <module>jar_artifact</module>
+                        <module>lombok</module>
                     </modules>
                     <excludedFiles>
-                        <excludedFile>.travis.yml</excludedFile>
+                        <excludedFile>.github/workflows/ci-build-next-java.yml</excludedFile>
```

Comment:
File seems to be exactly same file, do you need to exclude it in project-keeper?

Reply:
I added additional github token:
![image](https://user-images.githubusercontent.com/18247545/147224068-3e860783-6df0-41d8-a529-6b693be3b193.png)